### ISSUE TITLE
Feature Request: Sub Hired weight

### ISSUE THREAD

**Issue by TechJed** [CompositeLight/currentRMS-helper#75]
Would be great when trying to work out weights if you could look at the total sub hired weight, as if it is coming straight to site, i don't have to allow for it in the truck

**Comment by ClearsoundPro** [comment]
+1 for thisThere is a document you can use for this tooDan RiddClearsound Productions LtdMobile: 07976 627833Office: 01608 ***@***.***: www.clearsound.co.ukSent from my iPhoneOn 22 Sep 2024, at 14:52, TechJed ***@***.***> wrote:﻿
Would be great when trying to work out weights if you could look at the total sub hired weight, as if it is coming straight to site, i don't have to allow for it in the truck

**Comment by CompositeLight** [comment]
Added for testing in V1.31.19

**Comment by CompositeLight** [comment]
This seems to be working well. Next stage is to add a breakdown of weight for each sub-rent supplier.

**Comment by TechJed** [comment]
Ideal, could we add it to the order page too as it useful for PM's to work out how much weight is coming from the warehouse when looking at trucks etc

**Comment by CompositeLight** [comment]
I agree it would be helpful, I'll see what I can do in the future. The reason it's not there already / the difficulty is that in Detail View all the weight info for each line is already on the page in hidden cells (including for sub-rents, text items etc), so I can just scrape all that fro the page and add it up. Sadly it's not there in Order View, so makes it a lot harder to get the data; it'd either have to be a background page load of Detail View, or another API call(s) to pull it all out.

**Comment by CompositeLight** [comment]
Check V2.0.0 - currently available on the Development branch for testing. Sub-hire weights now shown in order view side bar, with a breakdown of weight per supplier.

**Comment by CompositeLight** [comment]
Text item bug fixed in 2.0.5. I also noticed Sales items weren't getting caught in the Stock Weight total - now fixed.
